lib/farscape/representor.rb

```diff
@@ -0,0 +1,109 @@
+require 'representors'
+require 'farscape/transition'
+require 'ostruct'
+
+module Farscape
+  class SafeRepresentorAgent
+    
+    include BaseAgent
+    
+    attr_reader :agent
+    attr_reader :representor
+    attr_reader :response
+
+    EMPTY_BODIES = { hale: "{}" } #TODO: Fix Representor to allow nil resources
+
+    def initialize(requested_media_type, response, agent)
+      @agent = agent
+      @response = response
+      @requested_media_type = requested_media_type
+      @representor = deserialize(requested_media_type, response.body)
+      handle_extensions
+    end
+    
+    %w(using omitting).each do |meth|
```

Comment:
what's so hard about using `def using` or `def omitting`.  Why the meta-programming?  

Oh, and if this trickery is removed, your yard doc will display those methods.